aiidalab_widgets_base/computational_resources.py

```diff
@@ -852,19 +852,26 @@ def _configure_computer(self, computer: orm.Computer):
 
         return True
 
+    def _run_things_on_existing_computer(self, label):
+        """Run things on an existing computer"""
+        try:
+            orm.Computer.objects.get(label=label)
+            for function in self._on_setup_computer_success:
+                function()
+        except common.NotExistent:
+            return False
+        else:
+            return True
+
     def on_setup_computer(self, _=None):
```

Comment:
I am a bit confused here. The function name is `on_setup_computer`, but the doc string says `Create a new computer`. What is the main purpose of this function?

Reply:
The main purpose of this function is when the "setup computer" button is clicked, setup (create) a new computer.

Reply:
> The main purpose of this function is when the "setup computer" button is clicked, setup (create) a new computer.

yep, that is correct.

Reply:
@yakutovicha thanks for the context. Okay, in that case it would be great to expand the doc string on this function to explain when this function is called. This widget is doing so many things that it is beneficial for things to be documented well.